document/CoreFormat.mdk

```diff
@@ -303,40 +303,40 @@ Suppose a tool is invoked with the command line
         /ConnectionString "Server=CorpServer;Database=Accounting;User Id=Admin;Password=S3cr#t" /input *.sql
 ```
 
-Then the value of the `invocationInfo` property might contain the redacted command line
+Then the value of the `invocation` property might contain the redacted command line
 
 ```
     [REMOVED]\DbScanner.exe /connectionString=[REMOVED] /input=*.sql 
 ```
 ~
 
 ~ IsoNote
-Redacting sensitive information from `invocationInfo` makes it more difficult to precisely reproduce
-an analysis run. The value of `invocationInfo` would have to be combined with information from another
+Redacting sensitive information from `invocation` makes it more difficult to precisely reproduce
+an analysis run. The value of `invocation` would have to be combined with information from another
 source to allow the run to be repeated.
 ~
 
-### `fileInfo` property { #runInfo-fileInfo }
+### `files` property { #run-files }
 
-A `runInfo` object should contain a property named `fileInfo` whose value is a JSON object,
+A `run` object should contain a property named `files` whose value is a JSON object,
 each of whose properties represents a file that was scanned in the course of the run.
 
-The `fileInfo` object should contain properties representing at least those files in which results were detected,
+Each property name in the `files` object shall be the URI of a scanned file.
+No two of these property names shall be equivalent as defined in &sect;[#uriProperties].
+
+Each property value in the `files` object shall be an array of `file` objects (&sect;[#file])
+which contain information about the file identified by the URI in the property name.
+
+The object specified by the `files` property should contain properties representing at least those files in which results were detected,
 but it may contain properties representing all scanned files (whether or not results were detected in those files),
 or for any subset of those files.
 
 ~ IsoNumberedNote { @IsoNumberedNote=1}
-`fileReference` objects contain information that is useful for viewers.
-Viewers will be able to provide the most information to users if the `fileInfo`
+`file` objects contain information that is useful for viewers.
+Viewers will be able to provide the most information to users if the `files`
 property is present and contains information for every file in which results were detected.
 ~
 
-Each property name in the `fileInfo` object shall be the URI of a scanned file.
-No two of these property names shall be equivalent as defined in &sect;[#uriProperties].
-
-Each property value in the `fileInfo` object shall be an array of `fileReference` objects (&sect;[#fileReference])
-which contain information about the file identified by the URI in the property name.
-
```

Comment:
I moved these two paragraphs up above the IsoNumberedNote, because the note referred to the `fileReference` (now `file`) object, which hadn't yet been mentioned.